lib/active_type/record_extension/inheritance.rb

```diff
@@ -48,6 +48,14 @@ def model_name
           end
         end
 
+        def module_parents
```

Comment:
Can we give this method any other name? We're already inheriting a class method `.module_parents` from `Module`, and I feel uneasy overriding this with custom behavior.

Reply:
`module_ancestors` feels the most correct, because that's what this method literally does, it provides all the module ancestors, one by one, all the way to `Object`